prime-router/src/test/kotlin/ReportTests.kt

```diff
@@ -192,14 +192,15 @@ class ReportTests {
 
         val oneReport = Report(
             schema = one,
-            values = listOf(listOf("a1", "b1"), listOf("a2", "b2")),
+            values = listOf(listOf("a1", "b1"), listOf("", "b2")),
             source = TestSource,
             metadata = metadata
         )
 
-        val oneDeidentified = oneReport.deidentify()
+        val oneDeidentified = oneReport.deidentify("TEST")
         assertThat(oneDeidentified.itemCount).isEqualTo(2)
-        assertThat(oneDeidentified.getString(0, "a")).isEqualTo("")
+        assertThat(oneDeidentified.getString(0, "a")).isEqualTo("TEST")
+        assertThat(oneDeidentified.getString(1, "a")).isEqualTo("")
         assertThat(oneDeidentified.getString(0, "b")).isEqualTo("b1")
     }
```

Comment:
Suggest that you add the test case for `.deidentify("")` since empty string is the default value and a typical edge case.